src/utils/tasks/taskSkillsTree.goerli.ts

```diff
@@ -2,9 +2,7 @@
  * Skills Tree For the Goerli Network
  */
 
-import { ConsumableItem } from '~core/ItemsList';
-
-const taskSkills: ConsumableItem[] = [
+const taskSkills = [
   {
```

Comment:
You can remove the task skills lists for now. We can bring them back when we'll need them.